### PR TITLE
Format fix of Synopsis in manuals

### PR DESCRIPTION
**Pull request by tfendin**
Fixed issues where the synopsis was split in to several paragraphs, or when several command variants was garbled into one line.

### CODE REVIEW

**Comment by spacewander** [comment]
@tfendin 
Could you make a shorter title to avoid breaking in the middle?

**Comment by tfendin** [comment]
@spacewander 
Is this title better?

File: man/git-alias.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ git-alias(1) -- Define, search and show aliases
 
 ## SYNOPSIS
 
-`git-alias`
-`git-alias` &lt;search-pattern&gt;
-`git-alias` &lt;alias-name&gt; &lt;command&gt;
+`git-alias`
```
Comment (spacewander) [inline man/git-alias.md:6]: Is this trailing whitespace necessary?

Comment (tfendin) [reply man/git-alias.md:6]: Yes, that's is the markdown syntax for a line break.

Comment (spacewander) [reply man/git-alias.md:6]: Would it be better to use multiple newlines?

Comment (tfendin) [reply man/git-alias.md:6]: No, that would create an empty line between the different command variants. Which is not the style used by other commands in git-extras (git bulk for example) or in git-core (git stash for example). I'd think it was good if the same style was used in git-extras as in git-core etc.

**Review by spacewander — APPROVED** [review]

**Pull request merged**